integration-tests/bats/sql-spatial-types.bats

```diff
@@ -241,7 +241,7 @@ SQL
 
     run dolt sql -q "ALTER TABLE table1 MODIFY COLUMN p LINESTRING SRID 4326"
```

Comment:
can you add a BATS to round trip a `MULTIPOINT`, and other spatial types if we don't have one